<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaturaBilgileriFormu</class>
 <widget class="QDialog" name="FaturaBilgileriFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fatura Bilgileri</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../materials.qrc">
    <normaloff>:/icons/Icons/receipt_invoice.png</normaloff>:/icons/Icons/receipt_invoice.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>278</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_TTemizle">
      <property name="text">
       <string>Tümünü Temizle</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/trash-empty.png</normaloff>:/icons/Icons/trash-empty.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Kaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/Ok.png</normaloff>:/icons/Icons/Ok.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Cikis">
      <property name="text">
       <string>Çıkış</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/No.png</normaloff>:/icons/Icons/No.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_Adi">
      <property name="text">
       <string>Fatura No:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_Adi"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_Kodu">
      <property name="text">
       <string>Gecerlilik Süresi:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_Kodu"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Fatura Tarihi:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>107</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../materials.qrc">:/icons/Icons/receipt_invoice.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Fatura Ekle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../materials.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_Cikis</sender>
   <signal>clicked()</signal>
   <receiver>FaturaBilgileriFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Kaydet</sender>
   <signal>clicked()</signal>
   <receiver>FaturaBilgileriFormu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Kodu</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FaturaBilgileriFormu</receiver>
   <slot>degisiklikvarmi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Adi</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FaturaBilgileriFormu</receiver>
   <slot>degisiklikvarmi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>degisiklikvarmi()</slot>
 </slots>
</ui>
